pages/tutorials/3.0/system/time.md

```diff
@@ -33,9 +38,17 @@ sf::Int32 msec = time.asMilliseconds();
 float     sec  = time.asSeconds();
 ```
 
+Conversions with the C++ standard library's `std::chrono::duration` are supported in the form of a new constructor template as well as a conversion operator.
```

Comment:
Don't think we need to point out that it's a "new" constructor.  
The tutorials will likely be copied forward and new users wouldn't know what's already there.